src/components/include/security_manager/security_manager.h

```diff
@@ -70,6 +72,9 @@ class SecurityManager : public protocol_handler::ProtocolObserver {
     ERROR_INTERNAL = 0xFF,
     ERROR_UNKNOWN_INTERNAL_ERROR = 0xFE  // error value for testing
   };
+
+  enum ContextCreationStrategy { kUseExisting = 0, kForceRecreation };
```

Comment:
@CMaylone description required

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad